.travis.yml

```diff
@@ -140,4 +145,28 @@ jobs:
         - travis_wait make test-e2e-images
         - odo logout
 
+    # Run integration test on Kubernetes cluster    
+    - <<: *base-test
+      stage: test
+      name: "devfile catalog, create, push, watch command integration tests"
+      before_script:
+        # Download kubectl, which is required for using minikube
+        - curl -sLo kubectl https://storage.googleapis.com/kubernetes-release/release/v1.7.0/bin/linux/amd64/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+        # Download minikube and Set up Kubernetes cluster
+        - curl -sLo minikube https://storage.googleapis.com/minikube/releases/v0.25.2/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+        - mkdir -p $HOME/.kube $HOME/.minikube
+        - touch $KUBECONFIG
+        - sudo minikube start --vm-driver=none --kubernetes-version=v1.7.0
+        - minikube update-context
+        - "sudo chown -R travis: /home/travis/.minikube/"
```

Comment:
Remove unnecessary double quote

Reply:
> Remove unnecessary double quote

This is required over here as removing double quotes won't read sudo crown in travis.

Reply:
ohh, is it. 
Ok, then let it be like this. Thanks